internal/test/wmcb/wmcb_test.go

```diff
@@ -198,9 +232,8 @@ func (vm *wmcbVM) initializeTestBootstrapperFiles() error {
 	if err != nil {
 		return fmt.Errorf("unable to create remote directory %s: %v", remoteDir, err)
 	}
-
 	// Download and extract the kube package on the VM
-	err = vm.remoteDownloadExtract(kubeNode, remoteDir+"kube.tar.gz", remoteDir)
+	err = vm.remoteDownloadExtract(framework.pkgs[0], remoteDir+"kube.tar.gz", remoteDir)
```

Comment:
Can you loop over the slice instead of passing on the slice along with indexes. In fact do you think, if we have a `name` in pkgInfo{} struct to uniquely identify the item in the slice?

hint: You can also use a map.